packages/platform/atoms/lib/getQueryParam.ts

```diff
@@ -0,0 +1,14 @@
+const getQueryParam = (paramName: string, defaultParam?: "setup") => {
+  if (typeof window !== "undefined") {
+    const currentUrl = new URL(window.location.href);
+    const searchParams = currentUrl.searchParams;
+
+    const paramater = searchParams.get(paramName);
+
+    return paramater;
+  }
+
+  return defaultParam;
```

Comment:
If we are not running this on browser (window is undefined) and don't pass defaultParam, why do we return "setup" - like what does "setup" mean?

Reply:
`setup` is the name of the default tab that we display in the event-type settings

Reply:
Then we should somehow denote that the getQueryParam is for event-type settings, because what if it is used somewhere else? Then having "setup" as a default query param seems random.

Reply:
What if we make the defaultParam prop as not optional? that way it won't be limited to just event-type settings right?

Reply:
Idk i would just remove defaultParam and if whatever param is requested return undefined and then its up to the user to decide what to do